packages/dds/task-manager/.gitignore

```diff
@@ -0,0 +1,52 @@
+# Compiled TypeScript and CSS
+dist
+lib
+
+# Babel
+public/scripts/es5
+
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+.cache-loader
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+node_modules
+
+# Typings
+typings
+
+# Debug log from npm
+npm-debug.log
+
+# Code coverage
+nyc
+.nyc_output/
+
+# Chart dependencies
+**/charts/*.tgz
+
+# Generated modeuls
```

Comment:
nit: typo

Reply:
Will fix here, unfortunately it's because I copied from another package and it looks like this typo is also present in 101 other packages.